module-bsp/board/rt1051/common/fsl_drivers/CMakeLists.txt

```diff
@@ -66,6 +66,11 @@ target_sources(fsl
         fsl_wdog.h    
 )
 
+target_compile_definitions(fsl 
+    PRIVATE 
+    LPI2C_WAIT_TIMEOUT=500000
```

Comment:
can't it be declared in some header?

Reply:
so I'd add a comment about this configuration in our I2C wrapper - just to double-check that it's known for developers and easier for finding a place to change it:P

Reply:
Done